cmd/notify-mailer/main.go

```diff
@@ -145,6 +146,10 @@ func (m *mailer) run() error {
 			m.log.Debugf("skipping %q: out of target range")
 			continue
 		}
+		if err := policy.ValidEmail(address); err != nil {
```

Comment:
I think putting this check here is a good idea in general, do you think this will significantly reduce errors we see? It seems like the check is no stricter than what we previously had in the RA, so any emails that got past that check for new account/update will also get past this check.

Reply:
Yep, the check is exactly the same as what we had in RA. But the RA check has been refined over time, so there are still some old addresses that don't meet it.